src/main/java/mainclasses/TaskList.java

```diff
@@ -0,0 +1,126 @@
+package mainclasses;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+
+public class TaskList {
+    private List<Task> newStorage;
+
+    public TaskList(List<Task> initialisedStorage) {
+        this.newStorage = initialisedStorage;
+    }
+
+
+    /**
+     * Prints the task in a list format, includes description, task type and whether task is completed
+     * @param num int index of task in the list
+     * @param task the task to be printed out
+     */
+    public String printTask(int num, Task task) {
```

Comment:
Should the name of the method be "getTask" or "getTaskInfo"? This is because this method is not "printing" the task information and it is more to return the task information before it gets printed.